src/core/integrate.cpp

```diff
@@ -462,6 +469,21 @@ void integrate_vv(int n_steps, int reuse_forces) {
 
     if (check_runtime_errors())
       break;
+
+    // Check if SIGINT has been caught.
+    if (ctrl_C == 1) {
+      // Reset the flag.
+      ctrl_C = 0;
+
+      // This function simulates the effect of a SIGINT signal
+      // arriving — the next time PyErr_CheckSignals() is called,
+      // KeyboardInterrupt will be raised. It may be called without
+      // holding the interpreter lock.
+      set_py_interrupt = true;
```

Comment:
The comment above is nonsense because the function call it refers to was removed earlier.